source/nodejs/adaptivecards-designer/index.html

```diff
@@ -18,7 +18,10 @@
 
 	<script type="text/javascript">
 		window.onload = function() {
-            ACDesigner.GlobalSettings.previewFeaturesEnabled = false;
+            ACDesigner.GlobalSettings.enableDataBindingSupport = false;
```

Comment:
>     [](start = 5, length = 4)

mixed tabs and spaces

Reply:
(same for `previewFeatures.html`) -- feel free to add `.html` to `.editorconfig` for tabs

---
In reply to: [331603446](https://github.com/microsoft/AdaptiveCards/pull/3510#discussion_r331603446) [](ancestors = 331603446)